### PR TITLE
Fix/centos build (#2509)

### PR DESCRIPTION
**Pull request by a-zakir**

### CODE REVIEW

File: .github/workflows/centos7.yml [hunk 1]
```diff
@@ -29,135 +30,77 @@ jobs:
...
+          mkdir archive
+          docker cp $container_id:/workspace/Antares_Simulator/_build/archive/ archive/ 
+          docker rm $container_id
+
+      - name: Install gh if needed
+        if: ${{ env.IS_RELEASE == 'true' }}
+        run: |
+          wget https://github.com/cli/cli/releases/download/v2.52.0/gh_2.52.0_linux_amd64.rpm
+          rpm -i gh_2.52.0_linux_amd64.rpm
+          gh --version
```
Comment (flomnes) [inline .github/workflows/centos7.yml]: On the ubuntu runners, gh is installed by default
```suggestion
```

**Review by flomnes — COMMENTED** [review]
For ccache, it requires updating the online cache for persistency between runs.

File: .github/workflows/centos7.yml [hunk 2]
```diff
@@ -2,10 +2,11 @@ name: Centos7 CI (push and/or release)
 
 on:
   release:
-    types: [created]
+    types: [ created ]
   push:
     branches:
       - develop
+      - fix/centos-build
```
Comment (payetvin) [inline .github/workflows/centos7.yml]: remove this

**Review by payetvin — CHANGES_REQUESTED** [review]

File: .github/workflows/centos7.yml (hunk 2 above)
Comment (a-zakir) [reply .github/workflows/centos7.yml]: done

**Review by payetvin — APPROVED** [review]

**Pull request merged**